pkg/mimir/mimir.go

```diff
@@ -123,6 +124,7 @@ type Config struct {
 	Worker           querier_worker.Config           `yaml:"frontend_worker"`
 	Frontend         frontend.CombinedFrontendConfig `yaml:"frontend"`
 	IngestStorage    ingest.Config                   `yaml:"ingest_storage"`
+	BlockBuilder     blockbuilder.Config             `yaml:"block_builder" doc:"hidden"`
```

Comment:
isn't this part of ingest storage? i mean we use the same kafka config and it's rightly coupled with kafka just like ingest storage

Reply:
I think it's fair having a root leve config. Ingest storage is the config shared between component. This config is block-builder specific.

Reply:
+1 to what Marco says. For me, ingest-storage is a package, that services like ingester, query-frontend, ruler, and now block-builder use. I.e. block-builder is not a feature of ingest-storage, it's the opposite 🤷